introduction.md

```diff
@@ -1,2 +1,12 @@
+# Hello Mastering Github course!
+
 Hello codeschool mastering github course om nom nom nom!
 My favourite path so far has been Ruby Path.
+
+## Deadly Skills
+
+* Javascript / CoffeeScript
+* CSS / SASS
+* Ember.js
+* GIT (in a while!)
+* English fluency
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?